src/assets/server/rest/v1/schemas/tag/tag.json

```diff
@@ -2,14 +2,7 @@
   "$id": "tag.json",
   "definitions": {
     "id": {
-      "type": "string",
-      "pattern": "^[\\w\\d]*$",
-      "sanitize": "mongo"
-    },
-    "IDs": {
-      "type": "string",
-      "pattern": "^(([\\w\\d]*)(\\||[\\w\\d])*)?$",
-      "sanitize": "mongo"
+      "$ref": "common.json#/definitions/string-id"
```

Comment:
We are loosing a bit of accuracy here, as the pattern of "string-id" allows everything while tag IDs should be only letters an digit. I would let this one as it is, what do you think ?

Reply:
I didn't want to put too many constraints on string ids because you will have to create a new type in common to handle the string-tag-id and string-tag-ids in filters also.
But as you want :).

Reply:
I will tend to prioritize accuracy against constraints in the files, as users, one day, always mess up their environment :)
It's up to you, you're the boss :)